### ISSUE TITLE
Increase z-index for the Aladin Lite div in the object page

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-science-portal#605]
The new Aladin Lite v3 in full screen mode is not correctly displayed (the top is below the navbar) in an object page.

![Screenshot from 2024-03-25 23-55-07](https://github.com/astrolabsoftware/fink-science-portal/assets/20426972/beb66e7e-ccc3-4252-a435-caccc6644d27)

**Comment by karpov-sv** [comment]
I would say they changed something in the code as it definitely did not look that ugly just a couple of weeks ago. Really hope they will revert it back...
<img width="367" alt="image" src="https://github.com/astrolabsoftware/fink-science-portal/assets/6555548/c2a1c2cf-0f83-454d-81ed-f9c614eccca2">

**Comment by karpov-sv** [comment]
I propose the following fix - basically, downgrading from (just released) v3.3 to the previous version, and fixing it to avoid such unexpected changes in the future:

```diff
diff --git a/app.py b/app.py
index bd27639..017637b 100644
--- a/app.py
+++ b/app.py
@@ -39,7 +39,7 @@ external_stylesheets = [
 ]
 external_scripts = [
     '//code.jquery.com/jquery-1.12.1.min.js',
-    '//aladin.u-strasbg.fr/AladinLite/api/v3/latest/aladin.js',
+    '//aladin.u-strasbg.fr/AladinLite/api/v3/3.2.0/aladin.js',
     '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js',
 ]
```

**Comment by JulienPeloton** [comment]
thanks for the proposal, I fully agree. The new changes are interesting, but indeed it is not yet enough mature (e.g. a lot of superposition of objects) to be integrated.
